docs/providers/tencent/cli-reference/create.md

```diff
@@ -53,7 +53,7 @@ Most commonly used templates:
 - tencent-php
 - tencent-go
 
-**Note:** The templates will deploy the latest version of runtime by default. When you want to configure specific version of runtime, like `Node.js6`, `Python2.7` or `PHP5`, you have to configure the `runtime` property in `serverless.yml`.
+**Note:** The templates will deploy the latest version of runtime by default. When you want to configure specific version of runtime, like `Node.js18`, `Python3.10` or `PHP5`, you have to configure the `runtime` property in `serverless.yml`.
```

Comment:
Probably we should use Python 3.9 here, like in the other places.

Reply:
In any case, we can't add Python 3.10 here until after https://github.com/serverless/serverless/pull/11922 is merged.

Reply:
Good catch, I will change it to Python 3.9.

I will keep the other Python3.10 changes thought and I suggest we wait until that PR is merged. I will make sure to resolve any conflicts swiftly.